docs/resources/ec_deployment.md

```diff
@@ -63,12 +72,18 @@ The required `elasticsearch` block supports the following:
 
 ##### Topology
 
+To set up multi-node Elasticsearch clusters, single or multiple topology blocks can be set, each with a `instance_configuration_id` different set. This is particularly relevant for Elasticsearch clusters with Hot / Warm topologies, Machine learning, etc.
+
 The optional `elasticsearch.topology` block supports the following:
 
 * `instance_configuration_id` - (Optional) Instance Configuration ID from the deployment template. By default, it will use the deployment template default instance configuration, but it can be changed. See top level note on `regions and deployment templates`.
```

Comment:
```suggestion
* `instance_configuration_id` - (Optional) Instance Configuration ID from the deployment template. By default, it will use the deployment template default instance configuration, but it can be changed. See top level note on regions and deployment templates.
```